Removed PersistentConnection from ASP.NET Core SignalR (#9466)
aspnet/Docs: aspnetcore/signalr/version-differences.md

The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.

> ### PersistentConnection Removal

Reviewer: Keep Removal and lowercase it or omit it, your choice. It still exists in 4.x, so I think omitting "Removal" makes sense.  We didn't really "remove" it from Core, we just never added it.
```suggestion
### PersistentConnection
```

> ### PersistentConnection Deprecation

Reviewer 2: ```suggestion
### PersistentConnection deprecation
```

> ### PersistentConnection Deprecation

Reviewer: If the class doesn't exist in Core, it's not deprecated, it just doesn't exist and that's a difference between the two, not deprecation.

It's still viable in 4.x SignalR unless it's deprecated there in which case the deprecation comment is not about Core.

Author: Makes sense. Changed the terminology - let me know if further changes are needed. Thanks!

> In ASP.NET Core SignalR, the [`PersistentConnection`](https://docs.microsoft.com/en-us/previous-versions/aspnet/jj918097(v%3dvs.118)) class has been removed. 

Reviewer 2: ```suggestion
In ASP.NET Core SignalR, the [PersistentConnection](https://docs.microsoft.com/previous-versions/aspnet/jj919047(v%3dvs.118)) class has been removed. 
```
